cmd/uplink/cmd/setup.go

```diff
@@ -71,19 +71,37 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 
 	var override map[string]interface{}
 	if !setupCfg.NonInteractive {
-		_, err = fmt.Print("Enter your Satellite address: ")
+		_, err = fmt.Print(`
+Pick satellite to use:
+  [1] mars.tardigrade.io
+  [2] jupiter.tardigrade.io
+  [3] saturn.tardigrade.io
+Please enter numeric choice or enter satellite address manually [1]: `)
 		if err != nil {
 			return err
 		}
+		satellites := []string{"mars.tardigrade.io", "jupiter.tardigrade.io", "saturn.tardigrade.io"}
 		var satelliteAddress string
 		_, err = fmt.Scanln(&satelliteAddress)
 		if err != nil {
-			return err
+			// fmt.Scanln cannot handle empty input
+			satelliteAddress = satellites[0]
```

Comment:
we can detect an empty input just checking `err != nil && n == 0` where `n` is from `n, err = fmt.Scanln(...`

We could use that to show an error saying that `satelliteAddress` is required and stop or loop over until one option is selected or a manual address is written.

Reply:
added conditions about `n == 0`

We can add loop later if needed.